sdcm/nemesis.py

```diff
@@ -429,14 +429,25 @@ def _add_and_init_new_cluster_node(self, old_node_ip=None, timeout=HOUR_IN_SEC *
         self.monitoring_set.reconfigure_scylla_monitoring()
         self.set_current_running_nemesis(node=new_node)  # prevent to run nemesis on new node when running in parallel
         new_node.replacement_node_ip = old_node_ip
+        disable_autocompaction = random.choice([True, False])
+        keyspaces = self.cluster.get_test_keyspaces()
         try:
             self.cluster.wait_for_init(node_list=[new_node], timeout=timeout)
+            if disable_autocompaction:
+                self.log.info(f'during bootstrap, autocompaction will be disabled for keyspaces {keyspaces}')
+                new_node.wait_jmx_up()
+                for keyspace in keyspaces:
+                    new_node.run_nodetool(sub_cmd='disableautocompaction', args=keyspace, ignore_status=True)
         except (NodeSetupFailed, NodeSetupTimeout):
             self.log.warning("Setup of the '%s' failed, removing it from list of nodes" % new_node)
             self.cluster.nodes.remove(new_node)
             self.log.warning("Node will not be terminated. Please terminate manually!!!")
             raise
```

Comment:
if error is raise, we don't put this back into place this should go into finally clause.

Reply:
it doesn't really matter, as if the setup fails for that node, it will not join the cluster, and will not be used, so, i don't think we should continue to send commands to it (considering this is a command that could fail/get stuck, so i wanted to avoid making changes to a "dead node")

Reply:
fair enough